### DISCUSSION TITLE
Ability to style like tailwind

### DISCUSSION THREAD

**Discussion by kishaningithub** [diegomura/react-pdf#2785] [Ideas]
I feel the [stylesheet api](https://react-pdf.org/styling) is less expressive compared to the tailwind style approach. Would be great to have a wrapper of sorts on top of the stylesheet api which gives the users this

**Comment by kishaningithub** [comment]
Continuing the conversation as a github issue - https://github.com/diegomura/react-pdf/issues/2786

**Comment by imyuanli** [comment]
Yes, I also believe that tailwind can write PDF styles faster
